packages/tools/components-package/nps.js

```diff
@@ -100,7 +100,7 @@ const getScripts = (options) => {
 			props: `node "${LIB}/copy-and-watch/index.js" --silent "src/**/*.properties" dist/`,
 		},
 		watch: {
-			default: `${tsCrossEnv} concurrently "nps watch.templates" "nps watch.typescript" "nps watch.api" "nps watch.src" "nps watch.styles" "nps watch.i18n" "nps watch.props"`,
```

Comment:
Was this removed accidentally?

Reply:
not accidentally, API generation has no watch mode and is not needed for the start command. The script was orobably a leftover from previous implementations.